src/components/website-management/website-form-components/customization/customization-components/CookieSettingDialog.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Switch,
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import Typography from "@mui/material/Typography";
+
+const CookieSettingsDialog = (props) => {
+  const { open, onClose } = props;
+
+  return (
+    <Dialog open={open} onClose={onClose} maxWidth="sm" fullWidth>
+      <DialogTitle sx={{ fontWeight: "bold" }}>Cookie Settings</DialogTitle>
+      <DialogContent>
+        <Typography>
+          When you visit our website, we may utilize cookies to store or
+          retrieve information on your browser. This information can pertain to
+          you, your preferences, or your device, and is primarily employed to
+          ensure that the website functions according to your expectations. By
+          using cookies, we aim to provide you with a seamless browsing
+          experience that aligns with your preferences.
```

Comment:
Let's check whether it's possible to add localization using react-i18n .

Reply:
noted